src/Controller/Os2formsPaymentController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\os2forms_payment\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\os2forms_payment\Helper\PaymentHelper;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Returns responses for OS2forms payment routes.
+ */
+class Os2formsPaymentController extends ControllerBase {
+
+  /**
+   * Constructor.
+   */
+  public function __construct(
+    private readonly PaymentHelper $paymentHelper
+  ) {
+
+  }
+
+  /**
+   * Create.
+   *
+   * @return Os2formsPaymentController
+   *   Return paymentController
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get(PaymentHelper::class),
+    );
+  }
+
+  /**
+   * Create payment.
+   *
+   * @return \Symfony\Component\HttpFoundation\Response
+   *   Returns response containing paymentId from Nets endpoint
+   */
```

Comment:
Tell a bit about where you got this code from, i.e. add a reference to the Nets Easy Payments documentation.

Reply:
Done